src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -273,6 +273,8 @@ data class IndividualUser(
   override fun canDeleteSubLocation(subLocationId: SubLocationId) =
       isAdminOrHigher(parentStore.getFacilityId(subLocationId))
 
+  override fun canDeleteSupportIssue(): Boolean = false
```

Comment:
Yes. I think the intention is that if a human needs to delete a support issue, they can do so on the Jira page itself, instead of going through an API. The only use case of deleting an issue is simply for cleaning up tests for now. 

Down the line we can maybe expose ways to transition a ticket to close or complete. Even then I am guessing there is not many use cases, perhaps for a user to cancel a support request.